docker-compose.yml

```diff
@@ -4,49 +4,109 @@ services:
         image: zquestz/bitcoin-unlimited
         container_name: bitcoind-regtest
         ports:
-          - "18332:18332"
+          - "18443:18443"
           - "28332:28332"
           - "50001:50001"
         volumes:
-          - ./data:/data
-          - ./bitcoin.conf:/data/bitcoin.conf
-          - ./bitcoin.conf:/root/.bitcoin/bitcoin.conf
+          - ./data/bitcoind:/data
+          - ./configs/bitcoin-unlimited.conf:/data/bitcoin.conf
+          - ./configs/bitcoin-unlimited.conf:/root/.bitcoin/bitcoin.conf
         entrypoint: "/entrypoint.sh"
         command: bitcoind
         networks:
           - bitcoin-bridging
 
-    nginx-regtest:
-        image: nginx:latest
-        container_name: nginx-regtest
+    # bitcoind-regtest:
```

Comment:
Duplication.

Reply:
They're not duplicated. The second (commented out) one is for Bitcoin Cash Node and Fulcrum indexer, whereas the first uncommented on is for Bitcoin Unlimited (and its included Electrs indexer). Currently BCHN and Fulcrum have a weird bug that causes them to reboot into 'mainnet' mode. I'm just leaving the comments here so I can continue to work on and debug it so we'll have support in the future for multiple nodes and indexers.